it/src/main/java/org/corfudb/universe/node/CorfuServer.java

```diff
@@ -0,0 +1,71 @@
+package org.corfudb.universe.node;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.ToString;
+import org.corfudb.runtime.view.Layout;
+import org.slf4j.event.Level;
+
+import java.time.Duration;
+
+import static lombok.EqualsAndHashCode.Exclude;
+
+/**
+ * Represent a Corfu server implementation of {@link Node} used in the {@link org.corfudb.universe.cluster.Cluster}.
+ */
+public interface CorfuServer extends Node {
+
+    @Override
+    CorfuServer deploy();
+
+    boolean addNode(CorfuServer server);
+
+    boolean removeNode(CorfuServer server);
+
+    ServerParams getParams();
+
+    Layout getLayout();
+
+    void connectCorfuRuntime();
+
+    enum Mode {
+        SINGLE, CLUSTER
+    }
+
+    enum Persistence {
+        DISK, MEMORY
+    }
+
+    @Builder
+    @Getter
+    @AllArgsConstructor
+    @EqualsAndHashCode
+    @ToString
+    class ServerParams implements NodeParams {
+        @Exclude
+        private final String logDir;
+        private final int port;
+        private final Mode mode;
+        private final Persistence persistence;
+        @Exclude
+        private final Level logLevel;
+
+        @Exclude
+        private final int workflowNumRetry;
+        @Exclude
+        private final Duration timeout;
+        @Exclude
+        private final Duration pollPeriod;
+        private final NodeType nodeType = NodeType.CORFU_SERVER;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this final field static too. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1170)